pkg/sql/opt/xform/testdata/external/tpce

```diff
@@ -1838,30 +1838,28 @@ project
  ├── immutable
  ├── key: (1)
  ├── fd: (1)-->(5,10,21,25,26,31,33,36-39)
- ├── inner-join (hash)
+ ├── left-join (merge)
```

Comment:
The main changes here and below are that we are now preferring lookup joins into fact tables `trade_type` and `exchange` which have 4 and 5 rows, respectively, according to the statistics. The number of lookups should be very small, so I doubt it affect performance meaningfully.